analyses/pypi/matcher.go

```diff
@@ -0,0 +1,51 @@
+package pypi
+
+// Matcher implements driver.Matcher Interface for Pypi
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/fabric8-analytics/cli-tools/analyses/driver"
+	"github.com/rs/zerolog/log"
+)
+
+var (
+	_ driver.StackAnalysisInterface = (*Matcher)(nil)
+)
+
+// Matcher is State Object for Pypi
+type Matcher struct {
+	FilePath string
+}
+
+// Ecosystem implements driver.Matcher.
+func (*Matcher) Ecosystem() string { return "pypi" }
+
+// DepsTreeFileName implements driver.Matcher.
+func (*Matcher) DepsTreeFileName() string { return "pylist.json" }
+
+// GeneratorDependencyTree creates pylist.json from requirements.txt
+func (m *Matcher) GeneratorDependencyTree(manifestFilePath string) string {
+	log.Debug().Msgf("Executing: Generate Pylist")
+	pylistGenerator := m.getPylistGenerator(filepath.Join("/tmp", "generate_pylist.py"))
```

Comment:
This will not work on Windows. Use os.TempDir or equvalent.

Reply:
Ack.